test/README.md

```diff
@@ -135,40 +135,100 @@ chmod u+x dotnet-install.sh
 ./dotnet-install.sh -c Current
 ~~~
 
-#### Cloud side setup for the tests
-The end-to-end tests require a number of cloud side resources i.e., IoTHub, Device Provisioning Service, and a Storage Container to be setup. This next section will walk you through how to setup the cloud resources.
+#### One time cloud resources setup for the tests
+The end-to-end tests require a number of azure cloud side resources i.e., IoTHub, Device Provisioning Service, and a Storage Container to be setup. This next section will walk you through how to setup the cloud resources.
 
-_TODO: Add instructions to do this setup using the CLI_
+##### Create a resource group
+If you don't already have a resource group, create one.
+Here is how you can create one using the CLI
+~~~ sh
+az group create --name {resource group name} --location {region}
+~~~
 
 ##### IoT Hub
 If you don't already have an existing IoT hub, create one. There is no special configuration required, except for making sure that your IoTHub is enabled for public access.
+Here is how you can create one using the CLI
+
+~~~sh
+# Create a free tier Iot hub
+az iot hub create --resource-group {resource group name} --name {IoT hub name} --sku F1 --partition-count 2

```

Comment:
@scb01 There can only be one free IoT hub per subscription, and the quotas are extremely tight so it won't take long to run into throttling. I wonder if recommending the free SKU here might generate support issues that are unrelated to the tests...

Reply:
I agree. While the intent was to provide a cost effective option, it may result in confusion due to the tight quotas. I will go without specifying the SKU or the partition count ( which defaults to the SKU S1 and 4 respectively).  The idea of removing the partition count is that decreases the concept count by one and reduces the cognitive burden.